recipes/holopy/COPYING

```diff
@@ -0,0 +1,203 @@
+ GNU GENERAL PUBLIC LICENSE
```

Comment:
Please drop this file. It is picked up from the downloaded source so we don't need it in the recipe.